src/main/java/pl/javastart/task/UniversityApp.java

```diff
@@ -13,7 +28,22 @@ public class UniversityApp {
      * @param lastName  - nazwisko prowadzącego
      */
     public void createLecturer(int id, String degree, String firstName, String lastName) {
+        Lecturer lecturer = findLecturer(id);
+        if (lecturer != null) {
+            System.out.println("Prowadzący z id " + id + " już istnieje");
+        } else {
+            lecturers[lectureCount] = new Lecturer(firstName, lastName, id, degree);
+            lectureCount++;
+        }
+    }
 
+    private Lecturer findLecturer(int id) {
+        for (int i = 0; i < lectureCount; i++) {
+            if (Objects.equals(lecturers[i].getId(), id)) {
```

Comment:
Metoda `Objects.equals()` służy do porównywania obiektów. Jeżeli porównujemy inty, to wystarczy ==